test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -643,6 +659,43 @@ def _assert_artifact_free(output, stray_artifacts):
             artifact, output.stdout
         ), f"Matched {artifact} in {output.stdout} while running {output.command}"
 
+def _test_sm_toolkit_and_ts_version(image, region):
+    """
+    @param image: ECR image URI
+    """
+
+    if "gpu" or "cpu" in image:
```

Comment:
This is always going to be true. You can rewrite it like this:
if "gpu" in image or "cpu" in image:

Reply:
Thanks Ken. this is fixed or no longer an issue